### PR TITLE
Move exporter json strings to python

### PR DESCRIPTION
**Pull request by efahy**

### CODE REVIEW

File: applications/views/documents.py [hunk 1]
```diff
@@ -11,24 +11,27 @@
...
 from lite_content.lite_exporter_frontend import strings
 from lite_forms.generators import form_page, error_page
 
 
 def get_upload_page(path, draft_id):
     paths = document_switch(path)
 
+    # Note the use of get_const_string for getting content strings by path.  Under notmal circumstances,
+    # we refer to strings directly, since they are Python class properties, however in this case the paths are
+    # built dynamically, based on party type so this method is necessary
```
Comment (BenAndrew-Kainos) [inline applications/views/documents.py]: I don't really like this. Could we instead extend the `document_switch` function to also return the title, back_link & description based on the path

File: templates/users/users.html [hunk 2]
```diff
@@ -6,12 +6,6 @@
 
 {% block body %}
 
-	{% if userDeactivated == "true" %}
-		<div class="information information-animated">
-			{% get_string 'users.deleted' %}
```
Comment (BenAndrew-Kainos) [inline templates/users/users.html]: Why was this removed? (just curious)

Comment (efahy) [reply templates/users/users.html]: Its quite interesting actually: Your missing string finder tool reported `Users.DELETED` as missing when I ran my scripts and converted the above.  And when I investigated, it turned out the users.deleted string never existed, even when the above code was added.  So I checked with Jan and he said to just ditch it

File: applications/views/documents.py (hunk 1 above)
Comment (efahy) [reply applications/views/documents.py]: `document_switch` extended as suggested 👍

File: applications/helpers/reverse_documents.py [hunk 3]
```diff
@@ -28,53 +29,72 @@ def document_switch(path):
...
             "download": get_end_user_document,
             "delete": delete_end_user_document,
             "homepage": "applications:end_user",
-            "strings": "end_user.documents",
             "has_description": False,
+            "attach_doc_description_field_string": strings.EndUser.Documents.AttachDocuments.DESCRIPTION_FIELD_TITLE,
+            "attach_doc_title_string": strings.EndUser.Documents.AttachDocuments.TITLE,
+            "attach_doc_return_later_string": strings.EndUser.Documents.SAVE_END_USER,
+            "delete_conf_back_link_string": strings.EndUser.Documents.AttachDocuments.BACK_TO_APPLICATION_OVERVIEW,
+            # .AttachDocuments.DESCRIPTION_FIELD_TITLE
```
Comment (BenAndrew-Kainos) [inline applications/helpers/reverse_documents.py]: Please remove comment

File: templates/users/users.html (hunk 2 above)
Comment (BenAndrew-Kainos) [reply templates/users/users.html]: Awesome :D
